src/shared/components/palette/case-flag/domain/case-flag-state.model.ts

```diff
@@ -1,8 +1,11 @@
+
```

Comment:
@tarikdogan-hmcts - please remove this extra empty line

Reply:
done